lib/nostrum/api.ex

```diff
@@ -1178,7 +1178,7 @@ defmodule Nostrum.Api do
 
   ```Elixir
   Nostrum.Api.get_guild_channels(81384788765712384)
-  {:ok, [%Nostrum.Struct.Channel{} | _]}
+  {:ok, [%Nostrum.Struct.Channel{guild_id: 81384788765712384} | _]}
```

Comment:
Off the top of my head I think there was another example like this in the pr. Could it be changed too?

Reply:
Found a couple that had errors in their examples. Fixed in latest commit.